src/bundle/Resources/views/content/content_edit/content_create.html.twig

```diff
@@ -0,0 +1,33 @@
+{% extends 'EzPlatformAdminUiBundle:content/content_edit:content_edit_base.html.twig' %}
+
+{% trans_default_domain 'content_create' %}
+
+{% block details %}
+    <h4 class="text-muted">{{ 'creating'|trans({'%contentType%': contentType.name})|desc('Creating - %contentType%') }}</h4>
+    <h1>
+        <svg class="ez-icon ez-icon-{{ contentType.identifier }}">
+            <use xlink:href="{{ asset('bundles/ezplatformadminui/img/ez-icons.svg') }}#{{ contentType.identifier }}"></use>
+        </svg>
+        {{ 'new_content_item'|trans({'%contentType%': contentType.name})|desc('New %contentType%') }}
+    </h1>
+
+    <div class="small">
```

Comment:
Maybe just `<small></small>` instead?

Reply:
Small is inline we need a block element here